### DISCUSSION TITLE
helmfile sync always invokes helm upgrade with --reset-values, need to skip it

### DISCUSSION THREAD

**Discussion by tarun-thakur** [helmfile/helmfile#868] [Questions] [answered]
Hi Team,
Thank you for your time.

I want to skip --reset-values flag while running 'helmfile sync' command which I see always invokes 'helm upgrade --install' with '--reset-values'.

I have used 'reuseValues: false' under the helmDefaults section in the helmfile.yaml

Kindly suggest.

**Comment by yxxhero** [comment]
`reuseValues: true`?

**Comment by yxxhero** [comment]
@tarun-thakur I don't know if I understand your meaning. do you want to use reuseValues mode when `helmfile sync`?

**Reply by tarun-thakur** [reply]
I do not want helmfile to invoke helm upgrade --install with --reset-values flag which it is doing all the time.

**Reply by tarun-thakur** [reply]
I am trying to install prometheus helm chart using my own values.yaml

When I run helmfile sync -f my-prometheus-helmfile.yaml --debug 
it runs below helm command:
`exec: helm upgrade --install prometheus prometheus-community/prometheus --version 15.13.0 --wait --create-namespace --values /var/folders/dz/qd0qmyqs7c702kys7928x1d40000gp/T/helmfile2032313949/prometheus-values-7dc4746848 --reset-values --history-max 10 --debug`

I just want helmfile to not include --reset-values when it internally invokes helm upgrade command.

**Comment by yxxhero** [comment]
@tarun-thakur do you need `--reuse-values` flag?

**Reply by tarun-thakur** [reply]
Can I skip both reuseValues and resetValues both?

**Reply by yxxhero** [reply]
no. you can't. I want to know your user case.

**Reply by tarun-thakur** [reply, marked as answer]
Actually I was trying to install prometheus via Helmfile and its installation was failing for some reason when I was doing it from helmfile, but helm install prometheus, was passing, I thought helmfile invoking helm install with --reset-values flag is causing that issue. But actual culprit was --wait flag I was giving in helmfile, somehow --wait flag was causing something.

I just tried without wait: true and it passed.

Thank you for your support. I appreciate it.

**Reply by yxxhero** [reply]
@tarun-thakur ok. thanks for your feedback. if you have more issues. please provide the log. so I can get more information.
